addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -389,10 +423,12 @@ var SlideDialog = Widget.extend({
                                 });
                                 // page_content may contain null characters
                                 pageContent = pageContent.replace(/\0/g, '');
-                                self.index_content = self.index_content + pageNumber + '. ' + pageContent + '\n';
+                                var index_content = self._formGetFieldValue('index_content') + pageNumber + '. ' + pageContent + '\n';
```

Comment:
This is weird.
"index_content" is not used and on the following line "inputVal" looks like it's not defined.
Should it be indexContent for both?